### PR TITLE
Sticky collective alert

### PR DESCRIPTION
**Pull request by orlando**
**What:** Make collective alert sticky

**Why:** Some users were unable to see it when coming from the campaign link

**How:** Add some code to the collective-alert component to make it sticky when pressing any Reply button in a Topic

## Media

https://www.loom.com/share/51b326fee410410497a3a84b21863588

**Mobile**
http://recordit.co/1shuqlUnhy

### CODE REVIEW

File: assets/javascripts/discourse/templates/components/topic-footer-buttons.hbs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<div class='topic-footer-main-buttons'>
```
Comment (duranmla) [inline assets/javascripts/discourse/templates/components/topic-footer-buttons.hbs:1]: What is this?
This is not related to the feature described right?

**Review by duranmla — APPROVED** [review]
Minor comment, besides, a kind reminder to make references to all changes you do in a PR, for instance here you also add missing i18n.

👌

File: assets/javascripts/discourse/templates/components/topic-footer-buttons.hbs (hunk 1 above)
Comment (orlando) [reply assets/javascripts/discourse/templates/components/topic-footer-buttons.hbs:1]: yes it is, this is the reply button at the bottom of the topic (the green one in the video).

By default is not shown when you don't have permissions to post. We need to show it so users know what do to post. 

I'll add the same tooltip we use in the post reply button.
